### ISSUE TITLE
Firefox does not allows any more to hide the nav bar

### ISSUE THREAD

**Issue by JanusDC** [vimperator/vimperator-labs#465]
##### Issue type:
Bug report

##### Version:
Vimperator:	3.12.0 (created: 2016/02/28 22:07:56)
Firefox:	Mozilla/5.0 (X11; Linux x86_64; rv:45.0) Gecko/20100101 Firefox/45.0

##### Description:
I use to hide the navbar with 
`set gui=nonavigation`
However, in the new Firefox 45 the sync button won't work any more when the navbar is hidden. Same problem with Pocket.

Here is my report about the problem on the sync button in the Firefox bug tracking system: [1257801](https://bugzilla.mozilla.org/show_bug.cgi?id=1257801)
And here is a similar report with respect to Pocket: [1227540](https://bugzilla.mozilla.org/show_bug.cgi?id=1227540)

In both cases the answer was that it is Vimperator who has to change the behaviour.

**Comment by timss** [comment]
Possibly related, or with some helpful tips:

- https://github.com/chrisaljoudi/uBlock/issues/549
- https://github.com/chrisaljoudi/uBlock/issues/720

Also, this is quite unfortunate indeed:

> Firefox no longer (since Australis / version 29) supports hiding the navigation toolbar. We got rid of the "open location" dialog and a number of other things that used to have to deal with it not being there.

**Comment by JanusDC** [comment]
I am using the [Hide navigation bar](https://addons.mozilla.org/en-US/firefox/addon/hide-navigation-bar/) addon toghether with vimperator since it shows the navbar when I put the mouse nearby, so it solves the problem with the icons. But I guess either a similar solution could be applied to vimperator, or a way to access the sync menu from the command line (which will actually be a great feature).

**Comment by HalosGhost** [comment]
Also note that [Classic Theme Restorer](https://addons.mozilla.org/en-US/firefox/addon/classicthemerestorer) is capable of hiding the Nav bar (along with returning a lot more customization that has been removed with Australis.

**Comment by JanusDC** [comment]
I have just found the best solution in one of the links given below:
`set gui=nonavigation`
and
`style -name ublick * #nav-bar * { visibility: visible; }`

It works like a charm :-)

**Comment by timss** [comment]
Cool, I was hoping that solution would work @JanusDC. 

Wouldn't really consider this issue resolved though, the style is just a dirty workaround. Since it seems to be broken out of the box "something" needs to be done so `gui=nonavigation` can be used without any extra config.

**Comment by JanusDC** [comment]
There is a drawback: each time I open a new tab with <ctrl+t>, the cursor is in the invisible hidden bar and I get insert mode. However, with :tabopen that does not happen.

**Comment by timss** [comment]
Right. I remember having the same thing happen with uMatrix/uBlock. There's a hacky workaround available for that too, although it's not perfect either:

    autocmd LocationChange about:newtab <ESC>

https://github.com/vimperator/vimperator-labs/issues/218#issuecomment-130776682
https://github.com/timss/dotfiles/commit/9ebab930ad02f65e65b0a6b235aa0a703a4dcef6

**Comment by HeidiSalami** [comment]
I'm using this in my userChrome.css
`#nav-bar, #urlbar-container, #search-container, #openLocation { display:none!important; }`
to hide the Nav-Bar. Works ever since the Aurora design started.
